### PR TITLE
Added a note about sampler probablities

### PR DESCRIPTION
**Pull request by urig**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -42,6 +42,9 @@ could set `logging.level.org.springframework.web.servlet.DispatcherServlet=DEBUG
 
 NOTE: Set `spring.application.name=myService` (for instance) to see the service name as well as the trace and span IDs.
 
+NOTE: By default, Sleuth https://cloud.spring.io/spring-cloud-static/spring-cloud-sleuth/2.0.0.M8/single/spring-cloud-sleuth.html#_sampling_in_spring_cloud_sleuth[emits only a portion of traces]. If you want to guarantee that all traces are emitted then set `spring.sleuth.sampler.probability = 1.0`.
```
Comment (adriancole) [inline README.adoc]: TODO: does someone know how to make a current link? (or otherwise ensure this doesn't go stale)

**Comment by urig** [comment]
Closed? Why? :(

**Comment by marcingrzejszczak** [comment]
Because it was already there, I just have forgotten to rename `percentage` to `probability` (https://github.com/spring-cloud/spring-cloud-sleuth/commit/f18b3369e03c7506e429e23794d3f581b51a3019). Also you modified a wrong file.

**Comment by urig** [comment]
Thanks Marcin. I respect your decisions but at the same time I really think this bit should be up there right at the beginning of Getting Started. Speaking from my experience as someone who wanted to try things out and failed on this.

**Pull request closed without merging**